docs/appimage.rst

```diff
@@ -3,14 +3,40 @@ Via AppImage
 
 `AppImage <https://appimage.org/>`_ is a format for distributing portable software on Linux without needing superuser permissions to install the application. It tries also to allow Linux distribution-agnostic binary software deployment for application developers, also called Upstream packaging. 
 
-AppImage allows xonsh to be run on any AppImage supported Linux distributive without installation and root access.
+In short the AppImage is the one executable file which contains the xonsh and Python. AppImage allows xonsh to be run on any AppImage supported Linux distributives without installation and root access.
+
+Get AppImage from Github
+------------------------
+You can get xonsh AppImage from Github and run it on your Linux without install:
```

Comment:
You can get the xonsh AppImage from GitHub and run it on your Linux machine without installing it:

Reply:
Thanks!